bitnami/thanos/templates/ruler/statefulset.yaml

```diff
@@ -109,10 +109,13 @@ spec:
             {{- .Values.ruler.extraFlags | toYaml | nindent 12 }}
             {{- end }}
           env:
-            - name: POD_NAME
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.name
+          - name: POD_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
```

Comment:
Please revert the changes to the POD_NAME env variable and use `nindent 12` instead

Reply:
I reverted the indentation change, but either works.